Userland/Libraries/LibGfx/ICC/Profile.cpp

```diff
@@ -1573,7 +1573,7 @@ ErrorOr<void> Profile::from_pcs(FloatVector3 const& pcs, Bytes color) const
             //
             // where redTRC^-1, greenTRC^-1, and blueTRC^-1 indicate the inverse functions of the redTRC greenTRC and
             // blueTRC functions respectively.
-            // If the redTRC, greenTRC, or blueTRC function is not invertible the behaviour of the corresponding redTRC^-1,
+            // If the redTRC, greenTRC, or blueTRC function is not invertible the behavior of the corresponding redTRC^-1,
```

Comment:
This is a spec comment (and a few other things look like spec comments).
The extract line 'behaviour' exists in: https://www.color.org/specification/icc1v43_2010-12.pdf

Reply:
(That's why I mentioned spec comments)

Reply:
Oh! I thought you meant the JS spec. Yup, good point, I'll revert that.